index.rst

```diff
@@ -200,20 +197,10 @@ Atmosphere Images
 
     You need to use sudo permissions with Docker. After using the sudo command,
     Atmosphere will ask you for your CyVerse password for security
-    purposes. You can remove the need to use ``sudo`` with Docker commands in the `Advanced Docker Setup <docker.html>`_ section. 
+    purposes. You can remove the need to use ``sudo`` with Docker commands in the `Advanced Docker Setup <docker.html>`_ section.
 
 ----
 
-*Next Steps:*
```

Comment:
The way "Next Steps" is used elsewhere in the Learning Center Quickstarts is to suggest other tutorials or quickstarts, e.g. after you use FASTQC you may want to look at the trimmomatic tutorial. In tutorials a "Description of results and next steps" explain what the next steps are and the reasoning behind them. 

I am not saying that there is nothing next to do, only that its redundant here, since you have a next button. Does that make sense?